compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/util/RepoUtils.java

```diff
@@ -95,16 +95,13 @@ public static Path getLibDir() {
      * @return settings object
      */
     public static Settings readSettings() {
-        if (settings == null) {
-            String tomlFilePath = RepoUtils.createAndGetHomeReposPath().resolve(ProjectDirConstants.SETTINGS_FILE_NAME)
-                                           .toString();
-            try {
-                settings = SettingsProcessor.parseTomlContentFromFile(tomlFilePath);
-            } catch (IOException e) {
-                settings = new Settings();
-            }
+        String tomlFilePath = RepoUtils.createAndGetHomeReposPath().resolve(ProjectDirConstants.SETTINGS_FILE_NAME)
+                                       .toString();
```

Comment:
Lets break it with period (.) being on the same line. For example, .resolve(ProjectDirConstants.SETTINGS_FILE_NAME).
toString();

Reply:
Fixed with fb2abec